lib/data/spatial/1-import.sh

```diff
@@ -83,4 +83,4 @@ _import "Download manually" "forest_types_sh" "forest_types_sh" "forest_types_sh
 
 _import "Download manually" "forest_types_so" "forest_types_so" "forest_types_so"
 
-_import "Download manually" "forest_types_gl" "forest_types_gl" "forest_types_gl"
+# _import "Download manually" "forest_types_gl" "forest_types_gl" "forest_types_gl"
```

Comment:
Glarus don't want their data on prod yet, so we remove it for now to build the tiles